config/routes.php

```diff
@@ -79,11 +79,11 @@
         ]
     );
     $builder->connect(
-        '/connect',
+        '/contact',
```

Comment:
We can do a Squash merge of the branch at the end, I don't see why one should use "amend" on a publicly announced commit as it invalidates the part of the history?

Reply:
I don't consider a PR to be a published part of the history, just a proposal for acceptance. In some cases, a squash merge is OK, but in other cases it would make sense to keep separate patches in the merge because they are (properly) doing separate useful steps as part of a larger change.